src-api/source/includes/_overview.md

```diff
@@ -64,7 +64,7 @@ curl -u <circle-token>: "https://circleci.com/api/..."
 curl "https://circleci.com/api/v1.1/me?circle-token=<circle-token>"
 ```
 
-You can add the API token using your [account dashboard](https://circleci.com/account/api). Both personal and project API tokens are supported for API v1, however, we recommend using personal API tokens at this time as project API tokens are **not** supported for API v2.
+You can add the API token using your [account dashboard](https://circleci.com/account/api). We recommend using **personal API tokens** at this time as project API tokens are **not** supported for API v2. Project API tokens are supported for a subset of API v1 endpoints.
```

Comment:
Technically "all but 1" is a subset, but maybe we should say "Project API tokens are supported by most API v1 endpoints".